.gitignore

```diff
@@ -0,0 +1,40 @@
+MANIFEST
+build
+dist
+_build
+docs/man/*.gz
+docs/source/api/generated
+docs/source/config.rst
+docs/gh-pages
+notebook/i18n/*/LC_MESSAGES/*.mo
```

Comment:
the notebook/* lines are specific to the jupyter/notebook repo and probably should be removed